python/tvm/relay/op/strategy/cuda.py

```diff
@@ -268,11 +268,24 @@ def conv3d_strategy_cuda(attrs, inputs, out_type, target):
                 wrap_topi_schedule(topi.cuda.schedule_conv3d_ncdhw_winograd),
                 name="conv3d_ncdhw_winograd.cuda",
                 plevel=5)
-    else: # layout == "NDHWC":
-        strategy.add_implementation(wrap_compute_conv3d(topi.cuda.conv3d_ndhwc),
-                                    wrap_topi_schedule(topi.cuda.schedule_conv3d_ndhwc),
-                                    name="conv3d_ndhwc.cuda",
-                                    plevel=10)
+    else:  # layout == "NDHWC":
+        strategy.add_implementation(
+            wrap_compute_conv3d(topi.cuda.conv3d_ndhwc),
+            wrap_topi_schedule(topi.cuda.schedule_conv3d_ndhwc),
+            name="conv3d_ndhwc.cuda",
+            plevel=10)
+        N, _, _, _, _ = get_const_tuple(data.shape)
+        _, _, _, CI, CO = get_const_tuple(kernel.shape)
+        if nvcc.have_tensorcore(tvm.gpu(0).compute_version):
```

Comment:
Could you please check both the target_name and nvcc.have_tensorcore to skip nvcc.have_tensorcore check when CUDA is not available? 
Please refer to [Unable to get started with Metal on macOS 10.13 with Radeon Pro 560 4096 MB](https://discuss.tvm.ai/t/unable-to-get-started-with-metal-on-macos-10-13-with-radeon-pro-560-4096-mb/6209/14) for details.

Reply:
Added check for cuda to both conv2d and conv3d strategy call.